src/nixpacks/environment.rs

```diff
@@ -16,19 +17,26 @@ impl Environment {
     pub fn from_envs(envs: Vec<&str>) -> Result<Environment> {
         let mut environment = Environment::default();
         for env in envs {
-            let v: Vec<&str> = env.split('=').collect();
-            if v.len() == 1 {
+            let matches = Regex::new(r"([^=]*)(?:=?)([^=]*)([\s\S]*)")
+                .unwrap()
+                .captures(env)
+                .unwrap();
```

Comment:
Should not `unwrap` here. Or anywhere below either.

Reply:
In my tests, there can never be less then 1 match so it's fine to unwrap.